src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -88,10 +89,16 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             return;
         }
 
-        const initialPage = targetAttachments.findIndex(compareImage);
+        let initialPage = targetAttachments.findIndex(compareImage);
+        const prevInitialPage = attachments.findIndex(compareImage);
```

Comment:
```suggestion
        let attachmentIndex = targetAttachments.findIndex(compareImage);
        const prevAttachmentIndex = attachments.findIndex(compareImage);
```

I feel that this naming is more correct. `findIndex` returns an index and not a page. The index is used as the page id.

Reply:
done